docs/tutorials/create-a-pipeline-that-blurs-faces-in-an-image.md

```diff
@@ -42,117 +42,189 @@ documentation to start them.
+    takes the pipeline's image as input and the second step takes the
+    [Pipeline](../reference/core-concepts/pipeline.md)'s image and the result of the
+    face detection as input.
+    
+    !!! note
+        The `identifier` field of each step is the name of the step in the
+        [Pipeline](../reference/core-concepts/pipeline.md). It is used to reference the
+        step in the `needs` and `inputs` fields.
+    
+    ### Post the pipeline
+    
+    Now that we have our [Pipeline](../reference/core-concepts/pipeline.md), we can
+    post it to the [Core engine](../reference/core-engine.md). To do so, go to the
+    FastAPI documentation of the running [Core engine](../reference/core-engine.md)
+    and use the `/pipelines` endpoint to post the
+    [Pipeline](../reference/core-concepts/pipeline.md) by clicking on the
+    `Try it out` button
+    
+    Simply copy the content of the `face-blur-pipeline.json` file and paste it in
+    the `body` field of the `/pipelines` endpoint and click on the `Execute` button.
+    
+    ![Post pipeline](../assets/screenshots/post-pipeline.png)
+    
+    You should receive a `200` response with the
+    [Pipeline](../reference/core-concepts/pipeline.md) you just posted.
+
+=== "Using the Core engine interface"
+
+    Go to the Webapp on your browser to the  "/createPipeline" page of the [Core engine](../reference/core-engine.md).
+    From here you can chose from all available services to add to the new pipeline. First select the Face detection 
+    service. You can use the use the searchbar and filters on the side to help you find it.
+
+    ![Select Face detection](../assets/screenshots/pipeline-creation-select-face-detection.png)
+    
+    Click the "Add" button of the service. This will a dd a new new node in the flow chart. 
+
+    ![Flow Chart service](../assets/screenshots/pipeline-creation-flow-chart-add-service.png)
+
+    Do the same with the Image Blur service. You should now have 4 nodes in the flow chart. You can click on the nodes to drag and 
+    move them around to rearrange their disposition.
+    
+
+    There are 3 types of nodes:
+
+    - Entry node. It defines the input files the pipeline needs, and their types. Each input file has a handle you can connect 
+    to a service to pass on the data as input for the service.
+
+    - Exit node. It is the last node of the pipeline. Only one service (one handle) can be connected to it. The service connected to it 
+    defines what the output of the pipeline as whole will be.
+
+    - Service node. Each one represents a distinct service. For each data input the service needs, there is a handle you can connect to a data source, either the output of another service or
+    the input files of the pipeline defined by the entry node.
+    
+
+    Click the "Add input file" button on entry node, select a name for your input, and select the types image/jpeg and image/png.
```

Comment:
"on the entry node"
"select the name `image` for your input" you need to force them to use the same parameters as yours so there will be no errors afterwards